starknet_py/tests/e2e/utils.py

```diff
@@ -79,3 +75,11 @@ async def get_deploy_account_transaction(
         constructor_calldata=[key_pair.public_key],
         max_fee=MAX_FEE,
     )
+
+
+def _get_random_private_key() -> int:
```

Comment:
Maybe let's name it something like `_get_random_private_key_unsafe` so it's not accidentally used in library code.

Reply:
It's already in `tests` module

Reply:
Yeah, but I think it'd still be possible to accidentally import that in your IDE. Review would most likely catch that but it is still risky.

Reply:
Okay, i'm down to change it ⬇️